pkg/server/server.go

```diff
@@ -97,11 +98,24 @@ func (s *Server) Start(errCh chan error) error {
 		return err
 	}
 
+	_, err = exec.Command("docker").Output()
```

Comment:
If the user does not use the local container registry, then docker is not a requirement.

We shouldn't preform this check on server start but rather on local container registry start.

Reply:
i will look into this and revert with an update